.github/workflows/push-master.yml

```diff
@@ -27,6 +27,28 @@ jobs:
       run: |
         yarn install --immutable | grep -v 'YN0013'
         yarn ${{ matrix.step }}
+  release:
```

Comment:
For the time being, can we make it run each merge? Just until we are happy. (Otherwise it will be a bit of a nightmare atm to trigger via release)

Reply:
It will be run, just not released (e.g. when doing release, execs should be added to assets while on a normal merge, we don't have anywhere to put the apps)